resources/lib/monitor.py

```diff
@@ -169,7 +169,7 @@ def onNotification(self, sender, method, data):
 
         elif method == 'GetUsers':
 
-            users = server['api'].get_users(data.get('IsDisabled', True), data.get('IsHidden', True))
+            users = server['api'].get_users()
```

Comment:
why change?

Reply:
It function signature was changed a while ago and upstream forgot to change this call apparently